lib/rucio/core/did.py

```diff
@@ -43,12 +43,13 @@
 
 if TYPE_CHECKING:
     from collections.abc import Callable, Sequence
```

Comment:
```suggestion
    from collections.abc import Callable, Iterator, Sequence
```

to be used for return type of `bulk_list_files`